src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -95,6 +95,10 @@
 	}
 }
 
+.ui-dropdownmenu-force-display {
+	display: block !important;
```

Comment:
I always afraid about using  !important
was there any issue when there was no !important?

Reply:
This is the only way to overwrite inline style. This class in not part of API and it is used only for check position of placeholder.